spec/requests/resources_spec.rb

```diff
@@ -7,8 +7,8 @@
 
       it 'returns all resources' do
         get '/resources'
-        returned_ids = response_json.map { |r| r['id'] }
-        expect(returned_ids).to match(resources.map(&:id))
+        returned_names = response_json['resources'].map { |r| r['name'] }
```

Comment:
I think we want to leave IDs on resources for now. Otherwise clients won't be able to identify particular resources in URLs.